independent-projects/resteasy-reactive/pom.xml

```diff
@@ -60,8 +60,8 @@
         <version.enforcer.plugin>3.2.1</version.enforcer.plugin>
         <version.surefire.plugin>3.2.5</version.surefire.plugin>
         <mutiny.version>2.5.8</mutiny.version>
-        <smallrye-common.version>2.1.2</smallrye-common.version>
-        <vertx.version>4.5.4</vertx.version>
+        <smallrye-common.version>2.3.0</smallrye-common.version>
```

Comment:
Is this strictly necessary? I'm not sure it's in line with what we have in the BOM so this is problematic.